app/routes/surveys/components/StaticSubmission.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+
+import React from 'react';
+import styles from './surveys.css';
+import type {
+  SubmissionEntity,
+  AnswerEntity
+} from 'app/reducers/surveySubmissions';
+import type { SurveyEntity, QuestionEntity } from 'app/reducers/surveys';
+import { RadioButton, CheckBox, TextArea } from 'app/components/Form';
+import { QuestionTypes } from '../utils';
+
+type Props = {
+  survey: SurveyEntity,
+  submission?: SubmissionEntity
+};
+
+const StaticSubmission = ({ survey, submission }: Props) => {
```

Comment:
Isn't it possible to reuse the answer form, by having all the fields readonly?

Reply:
Would probably generate more cluttercode than it would save I think. There is quite a bit of logic, styling, etc in the form that would need something like `readOnly ? <h1>...</h1> : somethingCompletelyDifferent`. One file would be serving two pretty different functions. That's why it better to separate them imo. But if you can find a clean way of combining them and proving me wrong I'm all ears, giving them the exact same styling would be nice.